docs/content/docs/deep/container-and-component-loading.md

```diff
@@ -25,225 +25,295 @@ as well as the finished diagram in [Appendix 1](#appendix-1).
-
-1. Creating the `ContainerRuntime` object
-2. Setting the `request` handler on the `ContainerRuntime`
-    - The `request` handlers are used to route requests through the `Container` (more on this later)
-    - The primary use is to get Components
-3. Providing a `ComponentRegistry` of Component Factories to the `ContainerRuntime`
-    - The `ComponentRegistry` is a `Map<string, () => Promise(IComponentFactory)>`
-    - Defines what Components can be created in the `Container`
-4. Creating the default `Component`
-
-![Image 6](/images/container-and-component-loading-6.jpg)
+TODO: is the `CodeLoader` the one that downloads the package and just provides a path to it to the `Container`?
 
-Containers can exist without Components but they are not very functional. The paradigm we've created is for the
-Container Developer (`instantiateRuntime` implementer) to create a default `Component`. The default `Component` is simply
-the first `Component` in the `Container`. Having a default `Component` allows the Hosting Application to make a `request`
-against the `Container` asking for the default `Component` without knowing what the default `Component` is (more on
-this later).
+![Image 4](images/container-and-component-loading-4.jpg)
 
-The default `Component` is created the same as every other `Component`. The only difference is that it is the first
-`Component` and created in the `instantiateRuntime` call as opposed to being created by another `Component` (also more on
-this later).
+At this point the `Container` has a pointer to the `ContainerRuntime` code, which it passes along as it creates a new
+`ContainerContext` **(5)** , and through the context executes the `instantiateRuntime()` method provided by the
+`ContainerRuntime` **(6.1)**.
 
-A `Component` is created by calling `createComponent("packageName")` on the `ContainerRuntime` **(6.2)**. The
-`ContainerRuntime` uses its `ComponentRegistry` to look for the entry of `"packageName"`. When it's found it creates a
-`ComponentContext` **(7)** and executes the corresponding `instantiateComponent` with the `ComponentContext` **(8.1)**.
-
-![Image 7](/images/container-and-component-loading-7.jpg)
-
-You might notice a lot of similarities between the `ContainerRuntime` creation flow and the `ComponentRuntime`
+The important thing to note here is that up until this point the Hosting Application and Fluid know nothing of the Fluid
+`ContainerRuntime` or the `DataStore` code.
+That code is provided after the `Container` is established and stored in the op stream.
+**This is powerful because it allows the Hosting Applications to load Containers without knowing the underlying code.**
+This is how Teams and Outlook can easily load the Fluid preview app `Container` and `DataStore`s.
+
+![Image 5](images/container-and-component-loading-5.jpg)
+
+The implementer of `instantiateRuntime()` is what we refer to as a "Container Developer".
```

Comment:
As I'm reading, I'm wondering if it would be useful to link to the API docs for some of these particular members. What do you think?